plugin/mmdb/fixtures/create_databases.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"log"
+	"net"
+	"os"
+
+	"github.com/maxmind/mmdbwriter"
+	"github.com/maxmind/mmdbwriter/inserter"
+	"github.com/maxmind/mmdbwriter/mmdbtype"
+)
+
+const (
+	cdir          = "81.2.69.142/32"
+	cityDBName    = "GeoLite2-City.mmdb"
+	countryDBName = "GeoLite2-Country.mmdb"
+)
+
+// Create new mmdb database fixtures in this directory.
+func main() {
```

Comment:
why is this needed when you check in the files anyway? And if needed it should probably be moved to a _test.go file

Reply:
Creating the database record was a bit boilerplate/painful so I felt keeping the boilerplate to recreate them in case further record fixtures (or databases) were needed, would be handy.

I can move them over a `_test.go` file, or even remove if you push for it.